File: comms/push-notifications/fetch-push-notification/fetch-push-notification.py
from twilio_comms import TwilioComms

client = TwilioComms(
    account_id="TWILIO_ACCOUNT_SID",
    auth_token="TWILIO_AUTH_TOKEN",
)

client.push_notifications.fetch(
    push_notification_id="comms_pushnotification_01h9krwprkeee8fzqspvwy6nq8",
)

File: comms/push-notifications/fetch-push-notification/fetch-push-notification.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->pushNotifications->fetch(
    'comms_pushnotification_01h9krwprkeee8fzqspvwy6nq8',
);
